pkg/ca/common_test.go

```diff
@@ -124,14 +124,8 @@ func TestVerifyCertChain(t *testing.T) {
 		t.Fatalf("expected error verifying cert with mismatched public keys: %v", err)
 	}
 
-	// Failure: Weak key
-	weakSubCert, weakSubKey, _ := test.GenerateWeakSubordinateCA(rootCert, rootKey)
```

Comment:
@cpanato Can we keep this test, but update the check for the error message?

Reply:
done